abs.yaml

```
@@ -0,0 +1,10 @@
+# disable tests on initial build due to circular dependency with pytorch cuda.
+build_parameters:
+  - "--suppress-variables"
+  - "--skip-existing"
+  - "--error-overlinking"
+  - "--no-test"
+
```

```suggestion

build_env_vars:
  ANACONDA_ROCKET_ENABLE_PY313 : yes
```
To enable a py313 build.